### ISSUE TITLE
Update ANTs tag?

### ISSUE THREAD

**Issue by ntustison** [ANTsX/ANTsPy#577]
Hey all,

I'm trying to debug a subtle KK issue and tracing it to a discrepancy between ANTs and ANTsPy.  I realized that the core ANTs used in ANTsPy is a bit [dated](https://github.com/ANTsX/ANTsPy/blob/master/scripts/configure_ANTsPy.bat#L21):

```
% git show 276cf0717945d3dd3c4298c607d9d6a788ba574e
commit 276cf0717945d3dd3c4298c607d9d6a788ba574e
Merge: 6eb6bf05 1b2b5412
Author: Philip Cook <cookpa@pennmedicine.upenn.edu>
Date:   Tue Mar 21 22:20:45 2023 -0400

    Merge pull request #1516 from ANTsX/parseWinPointers
    
    ENH: Parse windows pointers for write operations
```

Anybody have any thoughts?  Particularly, @cookpa  and @ncullen93  who've been doing maintenance, containerization, etc.

**Comment by cookpa** [comment]
Agree, we're due a new release and I think it would be good policy to bump the version when ANTs gets update, which often implies an ITK update also.

Could we increment to 0.5.0?

**Comment by ntustison** [comment]
Thanks @ncullen93  and @cookpa .  I would have to defer to y'all.  I imagine what I'm working on isn't as crucially tied to a specific ANTsPy version as what others are doing.

Specifically, @cookpa incrementing to 0.5.0 is fine with me.   Would you like to do the updating or would you rather I did it?  I don't think I've ever formally updated versions of any of the ANTsX packages.

**Comment by ncullen93** [comment]
Any good places to announce the new release? Neuroimaging forums, etc.. 

I'd also like to revamp the docs and move them to ants.dev/docs/antspy for a new release... I just haven't found a good tool yet to generate them in markdown.

**Comment by ntustison** [comment]
Yeah, promotion is something we've definitely struggled with.

**Comment by ntustison** [comment]
Okay, the KK issue I was having in ANTsPy where thickness values would occasionally blow up to ridiculous values does not exist in the KK version in ANTs.  I don't know if this is with the DiReCT image filter itself or something internal ITK but it would be good to update just for this reason alone.

**Comment by stnava** [comment]
yeah - I have seen that as well .... thanks for looking into

**Comment by ncullen93** [comment]
If anyone wants to update it then feel free, otherwise I can do it tomorrow.

**Comment by cookpa** [comment]
>Okay, the KK issue I was having in ANTsPy where thickness values would occasionally blow up to ridiculous values does not exist in the KK version in ANTs. I don't know if this is with the DiReCT image filter itself or something internal ITK but it would be good to update just for this reason alone.

I have a vague memory of seeing this and then thinking I fixed it with an explicit t=10 kwarg to the kelly_kapowski call, but I don't understand how that was needed because the default should have always been 10.

Anyway, I can tag a Github release today. I don't know if the last section of wheels.yml, sending things to PyPI, actually works though. But it should make wheels and attach them to the release.

**Comment by ntustison** [comment]
Yeah, I thought I remembered this issue, as well, but I thought it was much further back in the history then within the past year or so.  

I'm actually working on a [mouse-based analog to the cortical thickness pipeline](https://github.com/ANTsX/ANTsPyNet/blob/master/antspynet/utilities/mouse.py#L453-L457) and so the scale is obviously completely different but I haven't had  any success in determining any sort of correspondence with any of the parameters.  It was only when I tried to compare with the ANTs version that I noticed the differences in code although nothing in the diff output strikes me as being the culprit.
